### ISSUE TITLE
Vagrant branch needs sudo for /opt format and mounting tasks

### ISSUE THREAD

**Issue by mark-dce** [acozine/sufia-centos#5]
I'm trying to run the vagrant branch to set up a machine and it fails with

```
TASK: [vagrant | format volume for /opt] ************************************** 
failed: [default] => {"err": "mkfs.xfs: cannot open /dev/sdc: Permission denied\n", "failed": true, "rc": 1}
msg: Creating filesystem xfs on device '/dev/sdc' failed

FATAL: all hosts have already failed -- aborting
```

As far as I can tell, the "format volume for /opt" and "mount volume at /opt" need to be run with elevated privileges, but I don't see that happening anywhere in vagrant_housekeeping.yml, the vagrant/mail.yml, or the vagrant role in vagrant.yml.

Should I not be running the vagrant branch?

**Comment by ilessing** [comment]
Alicia and I ran into the same issue last friday.

**Comment by acozine** [comment]
Yes, and I didn't commit and check in the change. Fixed now in the vagrant branch with 945239156bd38f6fdc59df6b8e0bdd3bfc60137f.
